src/sdl2_compat.c

```diff
@@ -1172,6 +1179,11 @@ SDL_GetRevisionNumber(void)
     return 0;
 }
 
+DECLSPEC const char * SDLCALL
+SDL_GetRevision(void)
+{
+    return SDL2COMPAT_REVISION;
```

Comment:
Just asking.
Is there a need to include SDL3's `SDL_GetRevision` in the returned string?
So this function returns a concatenation of SDL2-compat's revision and SDL3's `SDL_GetRevision`.
I suppose this would be useful for bug reports.

Reply:
With `SDL2COMPAT_DEBUG_LOGGING` env var set, we hit the following in `LoadSDL3()`:
```
SDL3_Log("sdl2-compat 2.%d.%d, talking to SDL3 %d.%d.%d", SDL2_COMPAT_VERSION_MINOR, SDL2_COMPAT_VERSION_PATCH, v.major, v.minor, v.patch);
```
We can change that to report result of SDL3_GetRevision() maybe if really needed?

Reply:
I was just asking. Thanks for reminding me about `SDL2COMPAT_DEBUG_LOGGING`.